app/controllers/searchjoy/searches_controller.rb

```diff
@@ -39,7 +39,8 @@ def recent
     protected
 
     def set_search_types
-      @search_types = Searchjoy::Search.uniq.pluck(:search_type).sort
+      @search_types = Searchjoy::Search.uniq.pluck(:search_type)
+      @search_types = @search_types.reject(&:blank?).sort + @search_types.select(&:blank?)
```

Comment:
I dont know If it is _that_ important to include/ display Searches where the search_type is blank. Maybe I have mixed some requirements of my own project into this fix.